### ISSUE TITLE
[archive] ideas: CV research idea radar 2026-09-18

### ISSUE THREAD

**Issue by Jung-woojin** [Jung-woojin/cv-research-radar#3]
# CV 논문 아이디어 레이더 — 2026-09-18

최근 7일 CV/VLM 연구를 바탕으로 단순 논문 요약이 아니라 후속 연구 proposal 관점에서 정리했다. 이번 주에는 **representation vs readout**, **persistent identity/state**, **adaptive high-resolution compute**가 특히 강한 연구 축으로 보였다.

## 연결 가치가 높은 최신 연구

- **Understanding Dynamic Scenes at Gigapixel Scale: Wide-Area Spatio-Temporal Perception from UAVs** — 122MP UAV 영상에서 detection, tracking, VQA를 통합하고 latency-aware s-HOTA를 제안. tiny+small object 비중이 매우 높고 cross-frame identity reasoning이 여전히 취약함.
- **Decodable but Misrouted: Sparse Features Uncover a Readout Gap in Vision-Language Models for Harmful Meme Detection** — representation에 정보는 있는데 native output이 못 쓰는 readout gap을 정량화.
- **InfoTaxa: Information-Calibrated Label-Free Clustering for Fine-Grained Visual Taxonomy** — downstream method 한계와 representation information ceiling을 분리.
- **SVMemAgent: A Streaming Video Memory Agent for Query-Agnostic Online Frame Selection** — future query를 모르는 streaming video에서 제한된 memory에 어떤 frame을 남길지 학습.
- **Video-HolmesV2: Can MLLMs Reason with Spatio-Temporal Audio-Visual Evidence in Long Videos?** — answer뿐 아니라 supporting temporal/spatial/audio evidence를 함께 평가.
- **Using OCR Heads to Verbalize Image Semantics** — OCR용 attention head가 general semantic readout 역할도 한다는 causal 분석.
- **Hyper-RED: Scalable Event Pre-training via Semantic Hypergraph Distillation** — RGB→event transfer에서 token-wise 강제 정렬 대신 high-order relation distillation.
- **Learning Where to Focus: Self-Supervised Multi-Scale ViTs for Histopathology** — coarse attention으로 중요한 영역만 fine token으로 refine하는 adaptive-resolution 구조.

---

# 후보 1. Where Did the Tiny Object Go?
### 문제 정의
Small-object AP가 낮을 때 현재 연구는 곧바로 P2, attention, larger RF, new neck을 추가한다. 하지만 실패는 최소 두 종류다.

1. **Representation failure** — feature에 tiny-object 정보 자체가 없다.
2. **Readout failure** — 정보는 feature에 있는데 detection head가 제대로 읽지 못한다.

### 핵심 가설
```
Tiny AP gap
= representation gap
+ readout gap
```

P2/P3/P4/P5 각각에서 objectness, class, center, scale 정보가 얼마나 recoverable한지 probe하고 native head와 비교하면 병목을 분해할 수 있다.

### 방법
- Frozen backbone + FPN
- P2/P3/P4/P5 level별 low-capacity probe
- Objectness / class / center / scale probe
- Native head와 probe 성능 차이로 **Readout Gap Score** 정의
- gap이 큰 level만 lightweight readout router로 교정

### Baseline
- YOLOv8/YOLO11
- RT-DETR
- D-FINE
- standard FPN
- P2 추가
- linear probe / MLP probe / small conv probe

### 데이터셋
- VisDrone
- AI-TOD / AI-TOD-v2
- 가능하면 HARD external validation

### 평가
- AP50-95, APs, tiny recall
- Objectness recoverability
- Classification recoverability
- Localization recoverability
- Readout Gap Score
- RGS vs APs correlation

### 필수 ablation
- P2/P3/P4/P5
- probe capacity
- object-size bin
- pretrained vs scratch
- FPN fusion 전/후
- classification vs localization
- P2 추가 전/후 recoverability 변화

### 실패 가능성
- probe와 native head 차이가 거의 없으면 representation loss가 진짜 bottleneck이라는 negative result
- 강한 probe만 잘하면 probe가 새 task를 학습했을 수 있으므로 linear/low-capacity control 필수
- architecture별 failure mode가 다르면 오히려 architecture-dependent bottleneck이라는 finding

### 4~8주 MVP
1. YOLO backbone/FPN hook
2. level-wise objectness/class probe
3. localization/scale probe
4. object-size별 recoverability curve
5. causal ablation
6. RT-DETR/D-FINE cross-architecture
7. simple readout router
8. final cross-dataset validation

### 예상 contribution
1. Tiny-object failure를 representation-limited / readout-limited로 분해하는 diagnostic framework
2. FPN level × object scale별 정보 보존 정량화
3. Diagnostic-driven lightweight readout correction

### 제목 후보
- **Where Did the Tiny Object Go? Diagnosing Representation and Readout Bottlenecks in Object Detection**
- **Before Adding P2: Measuring Information Ceilings in Tiny Object Detection**
- **Tiny Objects Are Not Always Missing: Representation–Readout Gaps in Multi-Scale Detectors**

### 평가
- Novelty: 4.7/5
- Feasibility: 4.9/5
- Compute cost: 1.8/5
- Publication potential: 4.7/5

---

# 후보 2. Identity Is the Missing Modality
### 문제 정의
강한 image VLM이 cross-frame identity reasoning에서는 발전을 그대로 가져오지 못한다. frame 수를 늘리는 것보다 persistent object identity/state가 병목일 수 있다.

### 핵심 가설
```
frames + explicit object state
> frames only
```

특히 temporal distance가 길수록 ID, bbox, velocity, trajectory state가 효과적일 것.

### 구조
```
Video
 ├─ Keyframes
 └─ Detector + Tracker
      └─ Object State Memory
          ├─ ID
          ├─ bbox
          ├─ class
          ├─ velocity
          └─ trajectory
                ↓
        State Tokenizer
                ↓
      Sparse Frames + State Tokens
                ↓
               VLM
                ↓
     Answer + Evidence ID + Time
```

### Baseline
- single frame
- uniform 8/16/32 frame
- frame selector
- frames + boxes
- frames + class
- frames + GT ID
- frames + predicted ID
- state-only
- full state

GT ID upper-bound baseline이 핵심이다.

### 데이터셋
- HARD
- 자체 항만 CCTV

### 평가
- Answer accuracy / Macro-F1
- Temporal IoU
- Evidence Track F1
- Grounding IoU
- Hallucinated Evidence Rate
- IDF1 / HOTA
- token count / latency

### 필수 ablation
- GT vs predicted ID
- bbox only vs trajectory
- state frequency
- memory length
- text token vs learned token
- ID switch injection
- missing detection injection
- performance vs temporal distance

### 실패 가능성
- tracker error가 VLM error를 증폭
- strong VLM이 이미 implicit identity를 충분히 추론하면 explicit state gain이 작을 수 있음

### Novelty 방어
단순 YOLO + ByteTrack + VLM 조합이 아니라:
- GT identity upper bound
- predicted identity
- temporal horizon analysis
- evidence grounding
- tracking error → VLM error causal analysis

### 제목 후보
- **Identity Is the Missing Modality: Object-State Grounding for Video Language Models**
- **Do Video VLMs Need Persistent Identity?**
- **From Frames to Tracks: Explicit Object State for Long-Horizon Video Reasoning**

### 평가
- Novelty: 4.8/5
- Feasibility: 4.4/5
- Compute cost: 2.5/5
- Publication potential: 4.8/5

---

# 후보 3. Recall-Constrained Adaptive Resolution for Tiny-Object Detection

### 문제 정의
Ultra-high-resolution aerial image에서 full tiling은 비싸다. 그러나 coarse-to-fine selection은 coarse stage가 tiny object를 놓치면 irreversible miss가 생긴다.

### 핵심 formulation
```
minimize compute
subject to Tiny Recall >= R0
```

### 구조
```
Ultra-high-res frame
    ↓
Low-res Scout
 ├─ semantic score
 ├─ uncertainty
 ├─ micro-structure cue
 └─ motion prior
    ↓
Risk-Constrained Tile Router
    ↓
Selected High-Res Tiles
    ↓
High-Res Detector
    ↓
Cross-Tile Fusion
```

### 핵심 가설
pure semantic attention보다
- semantic saliency
- uncertainty
- high-frequency micro-cue
- motion

를 함께 쓰면 적은 tile로 tiny recall을 보존할 수 있다.

### Baseline
- full tiling
- uniform downsample
- SAHI
- random tile
- saliency tile
- uncertainty tile
- CRAFT-like attention selection
- proposed router

### 데이터셋
- HARD
- VisDrone
- DOTA-v2

### 지표
- AP50-95
- APtiny / APsmall
- Tiny Recall
- Tile Coverage Recall
- selected area %
- FLOPs
- end-to-end latency
- mAP/sec
- HARD에서는 s-HOTA

### 필수 ablation
- semantic / uncertainty / frequency / motion cue
- tile size
- overlap
- budget
- resolution
- hard constraint vs soft loss
- static vs adaptive threshold

### 실패 가능성
- coarse scout가 tiny evidence를 완전히 잃으면 router가 복구 불가
- router overhead가 full tiling 절감분을 상쇄 가능

### 제목 후보
- **Spend Pixels Where They Matter: Recall-Constrained Adaptive Resolution for Tiny Object Detection**
- **Risk-Aware High-Resolution Routing for Wide-Area Tiny Object Detection**
- **Not Every Pixel Needs P2: Adaptive Resolution under Tiny-Object Recall Constraints**

### 평가
- Novelty: 4.5/5
- Feasibility: 4.3/5
- Compute cost: 2.4/5
- Publication potential: 4.7/5

---

# 이번 주 최우선 추천

**Where Did the Tiny Object Go?**

이유는 method를 만들기 전에 2~4주 안에 현상을 죽이거나 살릴 수 있기 때문이다.

```
P2에 정보가 있음 + head가 못 읽음
→ readout 문제

P2에도 정보가 부족
→ representation 문제
```

어느 결과가 나와도 다음 연구 방향이 선명하다. 새 모듈을 먼저 만들기보다 **현재 representation이 무엇을 알고 있고 어디서 정보가 사라지는지 먼저 측정**하는 것이 이번 주 가장 강한 연구 전략이다.
